Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -404,14 +408,26 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
                 error_message="Failed to modify '{}'".format(issue)
             )
             if not response:
-                if assignee:
-                    issue._assignee = None
                 if opened is not None:
                     issue._opened = None
                 return None
 
         return self.add_comment(issue, why) if why else issue
 
+    def add_assignees(self, issue, assignees):
```

Comment:
This looks to be `set_assignees`, not `add_assignees`, right?

Reply:
Making a POST request to that endpoint doesn't override the assignees array, it just adds any assignees that already haven't been added. Not sure what the verb for that would be.

Reply:
`add` is right, in that case, just a bit surprised that GitHub's API makes this `add` rather than `set`. Documentation reference for others curious: [https://docs.github.com/en/rest/issues/assignees#add-assignees-to-an-issue](https://docs.github.com/en/rest/issues/assignees#add-assignees-to-an-issue)